src/services/components/index.js

```diff
@@ -47,7 +51,13 @@ export const getRegistry = (
       ...registry,
       [component.namespaceURI]: {
         ...registry[component.namespaceURI],
-        [component.localName]: component,
+        ...[component.localName, ...(component.localNameAliases || [])].reduce(
+          (acc, localName) => ({
+            ...acc,
+            [localName]: component,
+          }),
+          {},
+        ),
```

Comment:
The `getRegistry` function is quite complex. I would recommend extracting out the reducers into seperate functions and then creating tests for each instead of a single test for `getRegistry`.

Reply:
addressed in 112e4a0